doc/source/scripts/generate_all.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e  # exit on failure
```

Comment:
This is the most important part of the PR -- we exit the docs build if the scripts fail, which they will do if entries such as TNG are missing.